src/components/shared/Button/PaneToggle.css

```diff
@@ -8,10 +8,6 @@
   padding: 5px;
 }
 
-.toggle-button.vertical {
```

Comment:
I use horizontal mode, so not sure what removing this part changes. Is there insight you can share on this?

Reply:
The `vertical` class is not referring to the vertical position of Devtools. Actually, I don't know why is there.